pmp-frontend-app/src/components/ContactFormComponent.tsx

```diff
@@ -122,7 +122,7 @@ export default function ContactFormComponent(): ReactElement {
                             (<p>
                                 {(messageCharLimit - inputFields.message.length) + " characters left"}
                             </p>) : 
-                            <p className="error text-red-400">
+                            <p className="error text-error-content">
```

Comment:
I saw that error-content is white in our tailwind config. Is the error text visible with white text? If not we could try "error bg-error text-error-content" I think right?

Reply:
I see, thanks for spotting! I suggest we just use error text-error. Then the font will be red.
I change it quickly